Client/Ecosia/Extensions/BrowserViewController+Ecosia.swift

```diff
@@ -36,6 +36,13 @@ extension BrowserViewController: DefaultBrowserDelegate {
     }
 }
 
+extension BrowserViewController: WhatsNewViewDelegate {
+    func whatsNewViewDidShow(_ viewController: WhatsNewViewController) {
+        profile.prefs.setInt(1, forKey: PrefsKeys.WhatsNewPageSeen)
```

Comment:
I would suggest to not use profile.prefs and use Core.User instead

Reply:
💯 , as mentioned in my previous message, will definitely move it to the `Core.User` once this overall changes will be given a 👍